.github/workflows/bk-ci.yml

```diff
@@ -505,7 +509,8 @@ jobs:
       - name: Set up JDK 11
         uses: actions/setup-java@v4
         with:
-          java-version: 11
+          distribution: 'temurin'
+          java-version: 21
```

Comment:
Why upgrade the java version to jdk21? All the current CI's java version is jdk11

Reply:
@hangc0276 I previously thought that using JDK 21 for OWASP might be potentially faster and help expose more problems.

Additionally, considering that OpenJDK 11 will reach its end of life in October 2024, do we have plan to upgrade CI to jdk17?

Reply:
At least, we should keep the same with other CI's versions